lib/generators/setup_generator.rb

```diff
@@ -119,60 +132,77 @@ def copy_configs_and_locales
 
     # private config customization
     answer = prompt_for_value("Dev API Path", "https://cdrhdev1.unl.edu/api/v1")
-    config_replace("private", "api_path: https://cdrhdev1.unl.edu/api/v1", "api_path: #{answer}")
-
-    answer = prompt_for_value("Production API Path", "https://cdrhapi.unl.edu/v1")
-    config_replace("private", "api_path: https://cdrhapi.unl.edu/v1", "api_path: #{answer}")
-
-    answer = prompt_for_value("Dev IIIF Path", "https://cdrhdev1.unl.edu/iiif/2")
-    config_replace("private", "iiif_path: https://cdrhdev1.unl.edu/iiif/2", "iiif_path: #{answer}")
-
-    answer = prompt_for_value("Production IIIF Path", "https://cdrhmedia.unl.edu/iiif/2")
-    config_replace("private", "iiif_path: https://cdrhmedia.unl.edu/iiif/2", "iiif_path: #{answer}")
-
-    <<-HEREDOC
-Configuration files copied to config/private.example.yml, config/private.yml, and config/public.yml.
-Locale files copied to config/locales
-Updated with initial app customizations
-    HEREDOC
+    config_replace("private", "api_path: https://cdrhdev1.unl.edu/api/v1",
+                   "api_path: #{answer}")
+
+    answer = prompt_for_value("Production API Path",
+                              "https://cdrhapi.unl.edu/v1")
+    config_replace("private", "api_path: https://cdrhapi.unl.edu/v1",
+                   "api_path: #{answer}")
+
+    answer = prompt_for_value("Dev IIIF Path",
+                              "https://cdrhdev1.unl.edu/iiif/2")
+    config_replace("private", "iiif_path: https://cdrhdev1.unl.edu/iiif/2",
+                   "iiif_path: #{answer}")
+
+    answer = prompt_for_value("Production IIIF Path",
+                              "https://cdrhmedia.unl.edu/iiif/2")
+    config_replace("private", "iiif_path: https://cdrhmedia.unl.edu/iiif/2",
+                   "iiif_path: #{answer}")
+
+    <<-MSG.chomp
```

Comment:
👏 I like MSG over HEREDOC, the latter of which means almost nothing to me in terms of instinctual reaction